frontend/src/pages/Home.jsx

```diff
@@ -2,14 +2,30 @@ import React, { Component } from "react";
 import HomeContent from "../components/HomeContent/homeContent";
 import { connect } from "react-redux";
 import { bindActionCreators } from "redux";
-// import { Link } from "react-router-dom";
+import {
+  setLogin_email,
+  setLogin_password,
+  setRegister_email,
+  setRegister_password,
+  set_secondRegister_password,
+  setLogin_dialog,
+  setRegister_dialog,
+} from "../actions/Home";
 import logo from "../resources/logo.png";
 import { AppBar, Toolbar, Button, Typography } from "@material-ui/core";
+import TextField from '@material-ui/core/TextField';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import axios from "axios";
 
 import "../styles/Home.css"
 
-
 class Home extends Component {
+
```

Comment:
picky: using a de-structure like `const { propElement1, propElement2, ... } = this.props;` may help increase readability.

Reply:
this.props.xxx is for redux